content/docs/user-guide/troubleshooting.md

```diff
@@ -53,3 +53,16 @@ Unable to detect supported link types, as the
 [cache directory](/doc/command-reference/config#cache) doesn't exist. It is
 usually created automatically by DVC commands that need it, but you can create
 it manually (e.g. `mkdir .dvc/cache`) to enable this check.
+
+## Unable to acquire lock {#lock-issue}
```

Comment:
@harsh8398 looks great!!

The only thing that comes to my mind to change a bit is to expand on `If that is not the case, it usually means that DVC was terminated abruptly and manually ...` WDYT?

Otherwise looks great.